lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -134,8 +126,6 @@ def __init__(
             self._shared_axes["z"].join(self, sharez)
             self._adjustable = 'datalim'
 
-        auto_add_to_figure = kwargs.pop('auto_add_to_figure', True)
```

Comment:
I think we messed this up and did not switch the default to False for 3.5.  We may need to push the schedule back a version 😞 .

Reply:
👍